### PR TITLE
add orbit song 4

### PR DESCRIPTION
**Pull request by psugihara**
also allows songs to have diff intervals and adds fadeout for sample playback to avoid pops

### CODE REVIEW

File: src/sound.js [hunk 1]
```diff
@@ -144,12 +164,13 @@ export default class Sound {
       this.voices = parts.map(part => this.voiceFromFile(part[0]))
       
       // master output
-      const reverb = new Reverb(0.5)
-      reverb.wet.value = 0.15
+      this.reverb ||= new Reverb(0.5)
```
Comment (psugihara) [inline src/sound.js:167]: there was a little mem leak here on song change (addressed w compressor too)

**Comment by okwme** [comment]
big conflict here @psugihara  
is reverb 1.4 or 1.6???

**Comment by psugihara** [comment]
whoops hit merge :)

**Comment by psugihara** [comment]
that's the pan limit. It just pans the audio up to 70% left and right (1.4/2 == 70%) rather than hard panning.

**Pull request merged**